common/src/main/java/com/wynntils/core/overlays/objects/OverlayHolder.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.overlays.objects;
+
+import com.wynntils.core.config.objects.StorageHolder;
+import com.wynntils.core.features.properties.FeatureInfo;
+import com.wynntils.core.overlays.properties.Overlay;
+import java.lang.reflect.Field;
+
+public class OverlayHolder extends StorageHolder {
+    private final Overlay metadata;
+
+    public OverlayHolder(Object parent, Field field, Overlay metadata) {
+        super(
+                parent,
+                field,
+                field.getType(),
+                parent.getClass().getAnnotation(FeatureInfo.class).category(),
+                false);
+
+        this.metadata = metadata;
+    }
+
+    public Overlay getMetadata() {
+        return metadata;
+    }
+
+    public boolean getToggled() {
```

Comment:
I feel like these should be named "enabled" instead of "toggled" to match the language of the rest of the toggle-able aspects of the mod so far

Reply:
When making it I started realizing the it can be added to FeatureRegistry, and yeah I can change it all to enabled no problem. I'll work on it today.